server/src/sim/financialanalysis.py

```diff
@@ -48,7 +47,7 @@ def financialanalysis(D, postyear=2015, S=None, rerunmodel=False, artgrowthrate=
 
     # Get most recent ART unit costs 
     progname = 'ART'
-    prognumber = D['data']['meta']['progs']['short'].index(progname)
```

Comment:
@Ashalynd D['programs'] is a list of dictionaries now, and 'name' is set in loadworkbook.py, line 226 or so. It will be a problem if it will break all existing projects though...